heim-runtime/tests/linux.rs

```diff
@@ -0,0 +1,36 @@
+#[cfg(target_os = "linux")]
+use heim_runtime as rt;
+#[cfg(target_os = "linux")]
+use rusty_fork::rusty_fork_test;
+
+#[cfg(target_os = "linux")]
+rusty_fork_test! {
+    #[test]
+    fn test_default_procfs_root() {
+        let root = rt::linux::procfs_root();
+        assert_eq!("/proc", root);
+    }
+
+    #[test]
+    fn test_default_sysfs_root() {
+        let root = rt::linux::sysfs_root();
+        assert_eq!("/sys", root);
+    }
+}
+
+#[cfg(target_os = "linux")]
+rusty_fork_test! {
+    #[test]
+    fn test_nonstandard_sysfs_root() {
+        rt::linux::set_sysfs_root("/host/sys".to_string());
+        let root = rt::linux::sysfs_root();
+        assert_eq!("/host/sys", root);
+    }
+
+    #[test]
+    fn test_nonstandard_procfs_root() {
+        rt::linux::set_procfs_root("/host/proc".to_string());
+        let root = rt::linux::procfs_root();
+        assert_eq!("/host/proc", root);
+    }
+}
```

Comment:
I think each of these should run in its own process, otherwise, we can't be sure the process space isn't polluted at each of the tests.